src/zeta-convolver.cc

```diff
@@ -91,9 +91,9 @@ Convproc::Convproc (void)
 	, _nlevels (0)
 	, _latecnt (0)
 {
-	memset (_inpbuff, 0, MAXINP * sizeof (float*));
-	memset (_outbuff, 0, MAXOUT * sizeof (float*));
-	memset (_convlev, 0, MAXLEV * sizeof (Convlevel*));
+	memset (_inpbuff, 0, sizeof (_inpbuff));
+	memset (_outbuff, 0, sizeof (_outbuff));
+	memset (_convlev, 0, sizeof (_convlev));
```

Comment:
Here it would be useful to still have a reference to the size, perhaps like
```
memset (_inpbuff, 0, sizeof (_inpbuff)); // MAXINP
memset (_outbuff, 0, sizeof (_outbuff)); // MAXOUT
memset (_convlev, 0, sizeof (_convlev)); // MAXLEV
```
to better understand the constraints in `Convproc::configure` just below